audformat/core/table.py

```diff
@@ -593,6 +593,182 @@ def set(
         for idx, data in values.items():
             self.columns[idx].set(data, index=index)
 
+    def update(
+            self,
+            others: typing.Union[typing.Self, typing.Sequence[typing.Self]],
+            *,
+            overwrite: bool = False,
+    ) -> typing.Self:
+        r"""Update table with other table(s).
+
+        Table which calls ``update()``
+        to :ref:`combine tables <combine-tables>`
+        must be assigned to a database.
+        For all tables media and split must match.
+
+        Columns that are not yet part of the table will be added and
+        referenced schemes or raters are copied.
+        For overlapping columns, schemes and raters must match.
+
+        Columns with the same identifier are combined to a single column.
+        This requires that both columns have the same dtype
+        and if ``overwrite`` is set to ``False``,
+        values in places where the indices overlap have to match
+        or one column contains ``NaN``.
+        If ``overwrite`` is set to ``True``,
+        the value of the last table in the list is kept.
+
+        The index type of the table must not change.
+
+        Args:
+            others: table object(s)
+            overwrite: overwrite values where indices overlap
+
+        Returns:
+            the updated table
+
+        Raises:
+            RuntimeError: if table is not assign to a database
+            ValueError: if split or media does not match
+            ValueError: if overlapping columns reference different schemes
+                or raters
+            ValueError: if a missing scheme or rater cannot be copied
+                because a different object with the same ID exists
+            ValueError: if values in same position overlap
+            ValueError: if index is not conform to table index
```

Comment:
With conform you are referring to alike?

So maybe
```suggestion
            ValueError:  if level and dtypes of table indices do not match
```

Reply:
We had this wording in three other places, changed it there, too.

Reply:
Mhh, actually in the other places we have an index as input, so I will rephrase there to: `if level and dtypes of index does not match table index`.

Reply:
I copied the wording from the utils function, like `intersect`

Reply:
done